code42.json

```diff
@@ -1924,6 +1924,463 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
+        },
+        {
+            "action": "create case",
+            "identifier": "create_case",
+            "description": "Create a Code42 case",
+            "type": "investigate",
+            "read_only": false,
+            "parameters": {
+                "case_name": {
+                    "description": "Case name",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": false,
+                    "order": 0
+                },
+                "subject": {
+                    "description": "Subject of case (username)",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": true,
+                    "contains": [
+                        "email"
+                    ],
+                    "default": "",
+                    "order": 1
+
+                },
+                "description": {
+                    "description": "Description",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": false,
+                    "default": "",
+                    "order": 2
+                },
+                "assignee": {
```

Comment:
Should we add contains type for `assignee` and `subject`? `email` and `user name` ?